test/conformance/ptp/ptp.go

```diff
@@ -943,3 +980,96 @@ func ptpEventEnabled() bool {
 	}
 	return ptpConfig.Spec.EventConfig.EnableEventPublisher
 }
+
+func getNICInfo(pod v1core.Pod) map[string]string {
+
+	var ptpSupportedInterfaces []string = getPtpMasterSlaveAttachedInterfaces(pod)
+	var stdout bytes.Buffer
+
+	var ptpInterfaceNicMapping = make(map[string]string)
+
+	for _, interf := range ptpSupportedInterfaces {
+		PCIAddr := ""
+		var err error
+
+		Eventually(func() error {
+			stdout, err = pods.ExecCommand(client.Client, pod, PtpContainerName, []string{"readlink", "-f", fmt.Sprintf("/sys/class/net/%s", interf)})
+			if err != nil {
+				return err
+			}
+
+			if stdout.String() == "" {
+				return fmt.Errorf("empty response from pod retrying")
```

Comment:
When would this occur?

Reply:
comes from existing code again. Someone must have tested it before that I am not aware of.

Reply:
should be replaced by errors.New("empty response from pod retrying), to avoid the overhead of calling errors.New(Sprintf(...)).

Reply:
Looks like you missed one commit of fixing errorf with errors.New

Reply:
Done!